crates/lib3h/src/dht/mirror_dht.rs

```diff
@@ -448,6 +453,15 @@ impl MirrorDht {
             // Ask for its data and broadcast it because we want fullsync.
             DhtRequestToChild::HoldEntryAspectAddress(entry) => {
                 trace!("DhtRequestToChild::HoldEntryAspectAddress: {:?}", entry);
+                // if its shallow, ask for actual data
+                if entry.aspect_list.len() > 0 && entry.aspect_list[0].aspect.len() == 0 {
```

Comment:
Do we get shallow hold requests? I feel like we should make a different message type so we can assume which one is which and don't have to do a check like this.

Reply:
> Do we get shallow hold requests? 
Yes from the gossiping list. Only mirror DHT needs to behave differently so I don't think we need a different message type.